manual/source/Tutorial/DataStructures/Alignment/ScoringSchemes.rst

```diff
@@ -0,0 +1,187 @@
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp
+   :fragment: main
+
+The input sequences type ``TSequence`` and the :dox:`Align` object of type ``TAlign`` are defined and the two input sequences ``seq1 = "TELKDD"`` and ``seq2 = "LKTEL"`` together with the gap penalty are assigned. In this case we define only the gap value since the Blosum matrix will be used to score matches and mismatches.
+Then the sequences are associated with the alignment object.
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp
+   :fragment: init
+
+Now, we compute the global alignment function, providing as second parameter the tag referred to the Blosum62 matrix together with the gap costs.
+To do so, we simply call the function :dox:`globalAlignment` and give as input parameters the ``align`` object and the Blosum62 scoring scheme.
+The globalAlignment function returns the score of the best alignment, which we store in the ``score`` variable that is then printed together with the corresponding alignment.
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp
+   :fragment: alignment
+
+The output of a global alignment implementing the Blosum62 scoring function is as follows:
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp.stdout
+
+.. note::
+   As can be noted the output of this scoring scheme is completely different with respect to the output generated with the simple scoring scheme confirming that the scoring scheme choice is one of the most important step to achieve high quality alignments.
+
+Gap Evaluation
+--------------
+
+In the previous sections we proposed two simple code examples useful to highlight the differences between two scoring schemes capable to evaluate match and mismatch events. In this section we will see the three gap models, implemented in the SeqAn library, to evaluate the insertion and deletion events.
+
+Linear Gap Model
+^^^^^^^^^^^^^^^^
+
+The easiest is the Linear gap model that considers, for the alignment score computation, the gap length (g) giving the possibility to evaluate with different scores gaps of different sizes;
+
+.. image:: linear.png
+   :width: 160px
+   :align: center
+
+This gap model is chosen as standard when only a gap value is provided in the scoring function or when the two provided gaps have the same value. For instance, this gap model as been adopted during the alignment computation of the two proposed examples.
+
+Affine Gap Model
+^^^^^^^^^^^^^^^^
+
+It has been proven that the first amino acid or nucleotide inserted/deleted (identified as gap open) found during the alignment operations is more significant, from a biological point of view, than the subsequent ones (called gap extension), making the so called Affine Gap model a viable solution for the alignment of biomolecules.
```

Comment:
Maybe we need a citation about this..

"It has been proven that the first amino acid or nucleotide inserted/deleted (identified as gap open) found during the alignment operations is more significant .."